apps/web/src/views/Nft/market/Collection/MyNFTs/index.tsx

```diff
@@ -0,0 +1,49 @@
+import PageLoader from 'components/Loader/PageLoader'
+import { PageMeta } from 'components/Layout/Page'
+import dynamic from 'next/dynamic'
+import { NextRouter, useRouter } from 'next/router'
+import { useMemo } from 'react'
+import { useGetCollection } from 'state/nftMarket/hooks'
+import Header from '../Header'
+import Items from '../Items'
+
+const Traits = dynamic(() => import('../Traits'), {
+  loading: () => <PageLoader />,
+})
+// const Activity = dynamic(() => import('./Activity'), {
+//   loading: () => <PageLoader />,
+// })
+
+const getHashFromRouter = (router: NextRouter) => router.asPath.match(/#([a-z0-9]+)/gi)
+
+const Collection = () => {
```

Comment:
Is this being used?

Reply:
I just checked my side, and I don't think we are using this, I can confirm with Cesar! and will do update it if we don't need it